react-app-rewired/headers/csps/nameServices/yat.ts

```diff
@@ -0,0 +1,6 @@
+import type { Csp } from '../../types'
+
+// Simple proxy server to avoid cors and security error thrown by yat when calling the api directly
+export const csp: Csp = {
+  'connect-src': ['https://octopus-app-mkjlj.ondigitalocean.app'],
```

Comment:
going to slap a request changes on here so this doesn't get merged

this needs to be abstracted to an environment variable, e.g. https://github.com/shapeshift/web/blob/develop/react-app-rewired/headers/csps/defi/swappers/Osmosis.ts#L4